includes/Tasks/SetupSingleTask.php

```diff
@@ -3,10 +3,11 @@
 namespace Grrr\SimplyStaticDeploy\Tasks;
 
 use Simply_Static\Page;
-use Simply_Static\Plugin;
-use Simply_Static\Setup_Task;
 use Simply_Static\Task;
 use Simply_Static\Util;
+use Simply_Static\Plugin;
+use Simply_Static\Setup_Task;
```

Comment:
Why did you move these? Are we not using alphabetical order for these statements?